Fix warnings and errors in docs build

Building the docs with Doxygen currently raises a few warnings and errors:

```warning: argument 'interval' of command @param is not found in the argument list of AQNWB::BaseIO::createTimestampsAttributes(const std::string &path)
```

```
warning: The following parameters of AQNWB::NWB::ElectricalSeries::ElectricalSeries(const std::string &path, std::shared_ptr< BaseIO > io, const BaseDataType &dataType, const Types::ChannelVector &channelVector, const std::string &electrodesTablePath, const std::string &unit="volts", const std::string &description="no description", const SizeArray &dsetSize=SizeArray {0}, const SizeArray &chunkSize=SizeArray {1}, const float &conversion=1.0f, const float &resolution=-1.0f, const float &offset=0.0f) are not documented:
  parameter 'dataType'
  parameter 'channelVector'
  parameter 'unit'
  parameter 'dsetSize'
  parameter 'chunkSize'
  parameter 'conversion'
  parameter 'resolution'
  parameter 'offset'
```

```
warning: argument 'channels' of command @param is not found in the argument list of AQNWB::NWB::ElectrodeTable::ElectrodeTable(const std::string &path, std::shared_ptr< BaseIO > io, const std::string &description="metadata about extracellular electrodes")
```

```
 warning: The following parameter of AQNWB::NWB::NWBFile::createElectricalSeries(std::vector< Types::ChannelVector > recordingArrays, const BaseDataType &dataType=BaseDataType::I16) is not documented:
  parameter 'recordingArrays'
```

```
warning: argument 'recordingNumber' of command @param is not found in the argument list of AQNWB::NWB::NWBRecording::openFile(const std::string &rootFolder, const std::string &baseName, int experimentNumber, std::vector< Types::ChannelVector > recordingArrays, const std::string &IOType="HDF5")
/Users/oruebel/Devel/nwb/aq-nwb/src/nwb/NWBRecording.hpp:37: warning: The following parameters of AQNWB::NWB::NWBRecording::openFile(const std::string &rootFolder, const std::string &baseName, int experimentNumber, std::vector< Types::ChannelVector > recordingArrays, const std::string &IOType="HDF5") are not documented:
  parameter 'recordingArrays'
  parameter 'IOType'
```

```
/Users/oruebel/Devel/nwb/aq-nwb/src/nwb/base/TimeSeries.cpp:7: warning: The following parameters of AQNWB::NWB::TimeSeries::TimeSeries(const std::string &path, std::shared_ptr< BaseIO > io, const BaseDataType &dataType, const std::string &unit, const std::string &description="no description", const std::string &comments="no comments", const SizeArray &dsetSize=SizeArray {0}, const SizeArray &chunkSize=SizeArray {1}, const float &conversion=1.0f, const float &resolution=-1.0f, const float &offset=0.0f) are not documented:
  parameter 'dataType'
  parameter 'unit'
  parameter 'dsetSize'
  parameter 'chunkSize'
  parameter 'conversion'
  parameter 'resolution'
  parameter 'offset'
```

```
/Users/oruebel/Devel/nwb/aq-nwb/README.md:55: warning: Unexpected subsubsection command found inside section!
/Users/oruebel/Devel/nwb/aq-nwb/README.md:81: warning: unable to resolve reference to '/Users/oruebel/Devel/nwb/aq-nwb/.github/CODE_OF_CONDUCT.md' for \ref command
```

--- a/README.md
+++ b/README.md
@@ -41,7 +41,7 @@ multi-configuration generator:
 cmake --install build --config Release
 ```
 
-# Developing
+## Developing
 
 Build system targets that are only useful for developers of this project are
 hidden if the `aq-nwb_DEVELOPER_MODE` option is disabled. Enabling this
@@ -78,7 +78,7 @@ cmake --build --preset=dev --target=<name of the target>
 
 # Code of Conduct
 
-This project and everyone participating in it is govered by our [code of conduct guidelines](./.github/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
+This project and everyone participating in it is govered by our [code of conduct guidelines](https://github.com/NeurodataWithoutBorders/aqnwb/blob/main/.github/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
 
 # LICENSE
 

--- a/src/BaseIO.hpp
+++ b/src/BaseIO.hpp
@@ -291,7 +291,6 @@ public:
   /**
    * @brief Convenience function for creating timestamp related attributes.
    * @param path The location of the object in the file.
-   * @param interval Value is '1'
    * @return The status of the operation.
    */
   Status createTimestampsAttributes(const std::string& path);

--- a/src/nwb/NWBFile.hpp
+++ b/src/nwb/NWBFile.hpp
@@ -56,6 +56,9 @@ public:
   /**
    * @brief Create ElectricalSeries objects to record data into.
    * Created objects are stored in recordingContainers.
+   * @param recordingArrays vector of ChannelVector indicating the electrodes to 
+   *                        record from. A seperate ElectricalSeries will be 
+   *                        created for each ChannelVector
    * @param dataType The data type of the elements in the data block.
    * @return Status The status of the object creation operation.
    */
@@ -84,8 +87,7 @@ public:
   const std::string HDMFExperimentalVersion = "0.5.0";
 
   /**
-   * @brief Gets the TimeSeries object from the recording containers
-   * @param containerName The name of the timeseries group.
+   * @brief Gets the TimeSeries object from the recordingContainers
    * @param timeseriesInd The index of the timeseries dataset within the group.
    */
   TimeSeries* getTimeSeries(const SizeType& timeseriesInd);

--- a/src/nwb/NWBFile.hpp
+++ b/src/nwb/NWBFile.hpp
@@ -56,7 +56,7 @@ public:
   /**
    * @brief Create ElectricalSeries objects to record data into.
    * Created objects are stored in recordingContainers.
-   * @param recordingArrays vector of ChannelVector indicating the electrodes to 
+   * @param recordingArrays vector of ChannelVector indicating the electrodes to
    *                        record from. A separate ElectricalSeries will be
    *                        created for each ChannelVector
    * @param dataType The data type of the elements in the data block.

--- a/src/nwb/NWBFile.hpp
+++ b/src/nwb/NWBFile.hpp
@@ -57,7 +57,7 @@ public:
    * @brief Create ElectricalSeries objects to record data into.
    * Created objects are stored in recordingContainers.
    * @param recordingArrays vector of ChannelVector indicating the electrodes to 
-   *                        record from. A seperate ElectricalSeries will be 
+   *                        record from. A separate ElectricalSeries will be
    *                        created for each ChannelVector
    * @param dataType The data type of the elements in the data block.
    * @return Status The status of the object creation operation.

--- a/src/nwb/NWBRecording.hpp
+++ b/src/nwb/NWBRecording.hpp
@@ -38,7 +38,7 @@ public:
    * @param baseName The base name of the file (will be appended with
    * experiment number).
    * @param experimentNumber The experiment number.
-   * @param recordingArrays ChannelVector objets indicating the electrodes to use for 
+   * @param recordingArrays ChannelVector objects indicating the electrodes to use for
    *                        ElectricalSeries recordings 
    * @param IOType Type of backend IO to use
    */

--- a/src/nwb/NWBRecording.hpp
+++ b/src/nwb/NWBRecording.hpp
@@ -36,10 +36,10 @@ public:
    * @brief Opens the file for recording.
    * @param rootFolder The root folder where the file will be stored.
    * @param baseName The base name of the file (will be appended with
-   * experiment number).
+   *                 experiment number).
    * @param experimentNumber The experiment number.
-   * @param recordingArrays ChannelVector objects indicating the electrodes to use for
-   *                        ElectricalSeries recordings 
+   * @param recordingArrays ChannelVector objects indicating the electrodes to
+   *                        use for ElectricalSeries recordings
    * @param IOType Type of backend IO to use
    */
   Status openFile(const std::string& rootFolder,

--- a/src/nwb/NWBRecording.hpp
+++ b/src/nwb/NWBRecording.hpp
@@ -38,7 +38,9 @@ public:
    * @param baseName The base name of the file (will be appended with
    * experiment number).
    * @param experimentNumber The experiment number.
-   * @param recordingNumber The recording number.
+   * @param recordingArrays ChannelVector objets indicating the electrodes to use for 
+   *                        ElectricalSeries recordings 
+   * @param IOType Type of backend IO to use
    */
   Status openFile(const std::string& rootFolder,
                   const std::string& baseName,

--- a/src/nwb/base/TimeSeries.hpp
+++ b/src/nwb/base/TimeSeries.hpp
@@ -17,21 +17,18 @@ public:
    * @brief Constructor.
    * @param path The location of the TimeSeries in the file.
    * @param io A shared pointer to the IO object.
-   * @param dataType The data type to use for storing the recorded voltage signal
+   * @param dataType The data type to use for storing the recorded signal
    * @param unit Unit for the electrical signal. Must be "volts"
    * @param description The description of the TimeSeries.
    * @param comments Human-readable comments about the TimeSeries
-   * @param dsetSize Initial size of the main dataset. This must be a vector with two
-   *                 elements. The first element specifies the length in time and
-   *                 the second element must be equal to the length of channelVector
-   * @param chunkSize Chunk size to use. The number of elements must be two to 
-   *                 specify the size of a chunk in the time and electrode dimension
-   * @param conversion Scalar to multiply each element in data to convert it to 
-   *                 the specified ‘unit’
-   * @param resolution Smallest meaningful difference between values in data, stored 
-   *                 in the specified by unit
-   * @param offset Scalar to add to the data after scaling by ‘conversion’ to finalize
-   *                 its coercion to the specified ‘unit'
+   * @param dsetSize Initial size of the main dataset
+   * @param chunkSize Chunk size to use
+   * @param conversion Scalar to multiply each element in data to convert it to
+   *                   the specified ‘unit’
+   * @param resolution Smallest meaningful difference between values in data,
+   *                   stored in the specified by unit
+   * @param offset Scalar to add to the data after scaling by ‘conversion’ to
+   *               finalize its coercion to the specified ‘unit'
    */
   TimeSeries(const std::string& path,
              std::shared_ptr<BaseIO> io,

--- a/src/nwb/base/TimeSeries.hpp
+++ b/src/nwb/base/TimeSeries.hpp
@@ -17,8 +17,21 @@ public:
    * @brief Constructor.
    * @param path The location of the TimeSeries in the file.
    * @param io A shared pointer to the IO object.
+   * @param dataType The data type to use for storing the recorded voltage signal
+   * @param unit Unit for the electrical signal. Must be "volts"
    * @param description The description of the TimeSeries.
    * @param comments Human-readable comments about the TimeSeries
+   * @param dsetSize Intial size of the main dataset. This must be a vector with two
+   *                 elements. The first element specifies the lenght in time and
+   *                 the second element must be equal to the lenght of channelVector
+   * @param chunkSize Chunk size to use. The number of elements must be two to 
+   *                 specify the size of a chunk in the time and electrode dimension
+   * @param conversion Scalar to multiply each element in data to convert it to 
+   *                 the specified ‘unit’
+   * @param resolution Smallest meaningful difference between values in data, stored 
+   *                 in the specified by unit
+   * @param offset Scalar to add to the data after scaling by ‘conversion’ to finalize
+   *                 its coercion to the specified ‘unit'
    */
   TimeSeries(const std::string& path,
              std::shared_ptr<BaseIO> io,

--- a/src/nwb/base/TimeSeries.hpp
+++ b/src/nwb/base/TimeSeries.hpp
@@ -21,9 +21,9 @@ public:
    * @param unit Unit for the electrical signal. Must be "volts"
    * @param description The description of the TimeSeries.
    * @param comments Human-readable comments about the TimeSeries
-   * @param dsetSize Intial size of the main dataset. This must be a vector with two
-   *                 elements. The first element specifies the lenght in time and
-   *                 the second element must be equal to the lenght of channelVector
+   * @param dsetSize Initial size of the main dataset. This must be a vector with two
+   *                 elements. The first element specifies the length in time and
+   *                 the second element must be equal to the length of channelVector
    * @param chunkSize Chunk size to use. The number of elements must be two to 
    *                 specify the size of a chunk in the time and electrode dimension
    * @param conversion Scalar to multiply each element in data to convert it to 

--- a/src/nwb/ecephys/ElectricalSeries.hpp
+++ b/src/nwb/ecephys/ElectricalSeries.hpp
@@ -23,9 +23,9 @@ public:
    * @param electrodesTablePath Path to the electrodes table
    * @param unit Unit for the electrical signal. Must be "volts". 
    * @param description The description of the TimeSeries.
-   * @param dsetSize Intial size of the main dataset. This must be a vector with two
-   *                 elements. The first element specifies the lenght in time and
-   *                 the second element must be equal to the lenght of channelVector
+   * @param dsetSize Initial size of the main dataset. This must be a vector with two
+   *                 elements. The first element specifies the length in time and
+   *                 the second element must be equal to the length of channelVector
    * @param chunkSize Chunk size to use. The number of elements must be two to 
    *                 specify the size of a chunk in the time and electrode dimension
    * @param conversion Scalar to multiply each element in data to convert it to 

--- a/src/nwb/ecephys/ElectricalSeries.hpp
+++ b/src/nwb/ecephys/ElectricalSeries.hpp
@@ -18,22 +18,24 @@ public:
    * @brief Constructor.
    * @param path The location of the ElectricalSeries in the file.
    * @param io A shared pointer to the IO object.
-   * @param dataType The data type to use for storing the recorded voltage signal
+   * @param dataType The data type to use for storing the recorded voltage
    * @param channelVector The electrodes to use for recording
    * @param electrodesTablePath Path to the electrodes table
-   * @param unit Unit for the electrical signal. Must be "volts". 
+   * @param unit Unit for the electrical signal. Must be "volts".
    * @param description The description of the TimeSeries.
-   * @param dsetSize Initial size of the main dataset. This must be a vector with two
-   *                 elements. The first element specifies the length in time and
-   *                 the second element must be equal to the length of channelVector
-   * @param chunkSize Chunk size to use. The number of elements must be two to 
-   *                 specify the size of a chunk in the time and electrode dimension
-   * @param conversion Scalar to multiply each element in data to convert it to 
-   *                 the specified ‘unit’
-   * @param resolution Smallest meaningful difference between values in data, stored 
-   *                 in the specified by unit
-   * @param offset Scalar to add to the data after scaling by ‘conversion’ to finalize
-   *                 its coercion to the specified ‘unit'
+   * @param dsetSize Initial size of the main dataset. This must be a vector
+   *                 with two elements. The first element specifies the length
+   *                 in time and the second element must be equal to the
+   *                 length of channelVector
+   * @param chunkSize Chunk size to use. The number of elements must be two to
+   *                  specify the size of a chunk in the time and electrode
+   *                  dimension
+   * @param conversion Scalar to multiply each element in data to convert it to
+   *                   the specified ‘unit’
+   * @param resolution Smallest meaningful difference between values in data,
+   *                   stored in the specified by unit
+   * @param offset Scalar to add to the data after scaling by ‘conversion’ to
+   *               finalize its coercion to the specified ‘unit'
    */
   ElectricalSeries(const std::string& path,
                    std::shared_ptr<BaseIO> io,

--- a/src/nwb/ecephys/ElectricalSeries.hpp
+++ b/src/nwb/ecephys/ElectricalSeries.hpp
@@ -18,9 +18,22 @@ public:
    * @brief Constructor.
    * @param path The location of the ElectricalSeries in the file.
    * @param io A shared pointer to the IO object.
+   * @param dataType The data type to use for storing the recorded voltage signal
+   * @param channelVector The electrodes to use for recording
+   * @param electrodesTablePath Path to the electrodes table
+   * @param unit Unit for the electrical signal. Must be "volts". 
    * @param description The description of the TimeSeries.
-   * @param electrodesTablePath Location of the ElectrodesTable this time series
-   * was generated from
+   * @param dsetSize Intial size of the main dataset. This must be a vector with two
+   *                 elements. The first element specifies the lenght in time and
+   *                 the second element must be equal to the lenght of channelVector
+   * @param chunkSize Chunk size to use. The number of elements must be two to 
+   *                 specify the size of a chunk in the time and electrode dimension
+   * @param conversion Scalar to multiply each element in data to convert it to 
+   *                 the specified ‘unit’
+   * @param resolution Smallest meaningful difference between values in data, stored 
+   *                 in the specified by unit
+   * @param offset Scalar to add to the data after scaling by ‘conversion’ to finalize
+   *                 its coercion to the specified ‘unit'
    */
   ElectricalSeries(const std::string& path,
                    std::shared_ptr<BaseIO> io,

--- a/src/nwb/file/ElectrodeTable.hpp
+++ b/src/nwb/file/ElectrodeTable.hpp
@@ -19,7 +19,6 @@ public:
    * @brief Constructor.
    * @param path The path of the table.
    * @param io The shared pointer to the BaseIO object.
-   * @param channels The vector of channel numbers.
    * @param description The description of the table (default: "metadata about
    * extracellular electrodes").
    */

--- a/src/nwb/hdmf/table/DynamicTable.hpp
+++ b/src/nwb/hdmf/table/DynamicTable.hpp
@@ -64,7 +64,6 @@ public:
 
   /**
    * @brief Adds a column of element identifiers to the table.
-   * @param idDescription The description of the column.
    * @param elementIDs A unique pointer to the `ElementIdentifiers` dataset.
    * @param values The vector of id values.
    */

